low-level/sensors/impl/generic/cpu_fault_sensor.py

```diff
@@ -218,7 +218,7 @@ def _create_json_message(self, cpu, alert_type):
         epoch_time = str(int(time.time()))
 
         alert_id = self._get_alert_id(epoch_time)
-        host_name = socket.gethostname()
+        host_name = socket.getfqdn()
```

Comment:
getfqdn() is changing the host_name in alerts to private nw interface name, which is not what we need

>
>>> import socket
>>> socket.getfqdn()
'srvnode-3'
>>> socket.gethostname()
'sm17-r18.pun.seagate.com'
>>>

Reply:
There was a review comment by Ujjwal that we should use private hostname to link to these consumer groups we are introducing for Actuators, as public hostname may get changed in field. so in that getfqdn() is needed to get us private nw hostname, and not gethostname()
​
And obviously in sensor also we need to have same value. We will have to intimate CSM about this change, possibly by creating a jira ticket for them to check & assess